src/renderer/services/chain/types.ts

```diff
@@ -17,6 +17,22 @@ export type FeeLD = LiveData<Error, BaseAmount>
 export type FeesRD = RD.RemoteData<Error, Fees>
 export type FeesLD = LiveData<Error, Fees>
 
+export type SwapFeeRD = RD.RemoteData<Error, BaseAmount>
+export type SwapFeeLD = LiveData<Error, BaseAmount>
+
+export type SwapFeesLDS = {
+  source: SwapFeeLD
+  target: SwapFeeLD
+}
```

Comment:
Why do you still need this ^ type? Use `SwapFeesLD` everywhere, also to push it into `Swap` (instead of using nested RemoteData in `SwapFeesLDS`).

Reply:
arstifact